fpga.chart.py

```diff
@@ -253,10 +258,12 @@ def get_data(self):
 
                     for column in cursor.description:
                         name = fpga_key + '-' + column.name
+                        if self.fpga_count > 1:
+                            tot_name = 'fpga-total-' + column.name
                         if name in self.keys:
                             data[name] = row[columns[column.name]]
-
-
+                            if self.fpga_count > 1:
+                                data[tot_name] += row[columns[column.name]]
```

Comment:
How does this affect power/temp (should not appear in the totals, only data transfers)?

Reply:
These writes should only apply to the columns found with 'SELECT * FROM swarm64.get_fpga_stats()'.

The temp and power keys don't exists in the totals data.